app/components/containers/containersController.js

```diff
@@ -12,6 +12,7 @@ angular.module('containers', [])
     $scope.sortReverse = ($scope.sortType === sortType) ? !$scope.sortReverse : false;
     $scope.sortType = sortType;
   };
+  $scope.URLPublish = EndpointProvider.endpointURLPublish() || '0.0.0.0';
```

Comment:
This should not defaut to 0.0.0.0 here as the user may have configured its Docker daemon using the `--ip` flag. Docker doc reference: https://docs.docker.com/engine/reference/commandline/dockerd/#options

From what I understand, the only case where `EndpointProvider.endpointURLPublish()` would not be defined is when we create an endpoint using `EndpointService.createLocalEndpoint()`, right?